### ISSUE TITLE
MR-139: Close Button on Each Tab

### ISSUE THREAD

**Issue by kmscode** [mRemoteNG/mRemoteNG#154]
Merged from MR-139: https://mremoteng.atlassian.net/browse/MR-139

Received additional request via e-mail

Currently in mRemoteNG, an X is available at the top right corner of connection page to close them. Would it be possible to add an option, similar to Notepad ++ “Show close button on each tab”? This feature could be added in mRmoteNG, under the View menu. This would help with, not clicking the wrong close button when applications are maximized inside RDP sessions. Please see mockup below…

![image 2](https://cloud.githubusercontent.com/assets/16654335/19196370/4612056e-8c83-11e6-86be-15e5d431ed79.png)

**Comment by kmscode** [comment]
Depends on #155

**Comment by klonos** [comment]
This should be the default. Having the close button so far away from the actual tabs is a usability issue. That's how it used to be but things changed with the upgrade to the latest non-stable.

**Comment by Rusk85** [comment]
I am not sure what the current implementation is when you double-click a tab with your mouse pointer but I would like to see an option for enabling that behaviour. 
Should I create a separate issue/feature request for that in case this doesn't belong here.

**Comment by sparerd** [comment]
There is an option to make double clicking a connection tab close it. Default is that double clicking does nothing. 

Options -> Tabs & Panels -> Double clicking a connections tab closes it

**Comment by klonos** [comment]
Well, if we do move the close button in each tab, then double-click is free to be assigned some other functionality like duplicating the session (common behavior in most web browsers AFAIK).

**Comment by kmscode** [comment]
@klonos - there were never any close buttons on the tabs themselves. Not even in the 1.72 and older *NG releases.

#161 has been opened about reassigning the double click operation. I like that idea! 😄

**Comment by kmscode** [comment]
With the implementation of #155, the default theme includes close buttons on each tab. Closing.
